easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
I think pgi themselves distribute modulefiles and I think they have the
lowercase name. (Away from my computer right now)

So in that sense the official name is what they use there.

On Sun, Sep 13, 2015, 4:21 AM Kenneth Hoste <notifications@github.com>
wrote:

> In easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1833#discussion_r39345616>
> :
>
> > @@ -0,0 +1,22 @@
> > +name = 'pgi'
>
> You have full control over module names in EasyBuild, so using the
> official name PGI here doesn't mean you can't obtain a module named
> pgi/.... It does mean you'll get PGI/... by default though, that's true.
>
> A quick look at http://www.pgroup.com/ makes it clear to me that PGI is
> the official name; pgi is not mentioned at all on the main page (I didn't
> look beyond that).
>
> Personally, I'm not convinced that going with pgi here because most sites
> seem to use pgi as a module name is a good reason to diverge from our
> policy of using 'official' software names in easyconfig files, especially
> since eb supports using a custom module naming scheme to consistently
> make all of your modules lowercase if you want to.
> But, I don't want to turn this into a bikeshedding discussion either...
>
> @rjeschmi <https://github.com/rjeschmi>: thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1833/files#r39345616>
> .
>